### PR TITLE
Change var to const and let

### PR DESCRIPTION
**Pull request by garcai**
Change nytimes/index.html and nytimes/starter-page.html to change var to const and let. Also commented out unused variables submitBtn and playNav.

### CODE REVIEW

File: javascript/apis/third-party-apis/nytimes/index.html [hunk 1]
```diff
@@ -44,137 +44,40 @@ <h1>NY Times article search</h1>
...
-          pageNumber--;
-        } else {
-          return;
-        }
-        fetchResults(e);
-      };
+    // Defining a baseURL and key to as part of the request URL
+
+const baseURL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json';
+const key = 'INSERT-YOUR-API-KEY-HERE';
```
Comment (MrHeer) [inline javascript/apis/third-party-apis/nytimes/index.html]: Why not indented

Comment (garcai) [reply javascript/apis/third-party-apis/nytimes/index.html]: Sorry copied from another local virtual host where I was testing. Fixed and did another PR

Comment (garcai) [reply javascript/apis/third-party-apis/nytimes/index.html]: The new pull request is named nytimes-var does it matter it is a different name?

Comment (garcai) [reply javascript/apis/third-party-apis/nytimes/index.html]: I fixed the indenting, hopefully I haven't messed up again

**Comment by chrisdavidmills** [comment]
@garcai hrm, when I try running your code, it says "json.response" isn't defined in line 112. So something is going wrong with the fetch. Did you get yours to work properly?

**Comment by garcai** [comment]
I don't know how that happened. It works now and I did another pull request. Sorry

**Comment by chrisdavidmills** [comment]
Verified; this now works fine. Thanks a lot for your help here @garcai !

**Pull request merged**